web/client/epics/featuregrid.js

```diff
@@ -1052,5 +1062,6 @@ export default {
     virtualScrollLoadFeatures,
     replayOnTimeDimensionChange,
     hideFeatureGridOnDrawerOpenMobile,
-    hideDrawerOnFeatureGridOpenMobile
+    hideDrawerOnFeatureGridOpenMobile,
+    deactivateSyncWmsFilterOnFeatureGridClose
```

Comment:
can you try to remove the export default in the featuregrid epic and use 

import * as epics in the Feature Editor plugin

and do a quick round of tests of feature grid behaviour?